game/controllers.cc

```diff
@@ -109,7 +109,7 @@ struct ControllerDef {
 		if (ev.source.type != sourceType) return false;
 		if (!deviceMinMax.matches(ev.source.device)) return false;
 		if (!channelMinMax.matches(ev.source.channel)) return false;
-		if (!deviceRegex.empty() && !regex_search(devName, deviceRegex)) return false;
```

Comment:
same here

Reply:
Like i said on discord. The std:basic_regex has no empty method anymore see this paper: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2003/n1507.htm

If a std::basic_regex (std::regex is derived from it) is filled with an empty string it will not match anything. So it wouldn't matter much